confgen/api_internal.go

```diff
@@ -954,10 +982,18 @@ func fetchVolumeInfo(confMap conf.ConfMap) (whoAmI string, localVolumeGroupMap v
 
 	for volumeGroupName, volumeGroup = range globalVolumeGroupMap {
 		if whoAmI == volumeGroup.PrimaryPeer {
-			localVolumeGroupMap[volumeGroupName] = volumeGroup
 			for volumeName, volume = range volumeGroup.VolumeMap {
 				localVolumeMap[volumeName] = volume
 			}
+
+			var shared bool
```

Comment:
I don't think declaring a local scope var shared here follows the pattern in the codebase

Reply:
It is correct that this doesn't follow the pattern in the codebase.

Whether its a good pattern is a different question that i won't debate here.  i'll change this.